src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a source of contact information of a Person.
+ */
+public class Contact {
+
+    protected boolean isPrivate;
+    protected String value;
+
+    protected Contact(boolean isPrivate) {
+        this.isPrivate = isPrivate;
+    }
+
+    public Contact() {
+    }
+
+    @Override
+    public int hashCode() {
+        return value.hashCode();
+    }
+
+
+    public boolean isPrivate() {
+        return isPrivate;
+    }
+
+    public String toString() {
+        return value;
+    }
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.